src/libs/AxelarQueryAPI.ts

```diff
@@ -310,10 +310,18 @@ export class AxelarQueryAPI {
       const ethTokenPrice = Number(ethereumToken.token_price.usd);
       const ethToSrcTokenPriceRatio = ethTokenPrice / srcTokenPrice;
 
-      const actualL1ExecutionFee = l1ExecutionFee
+      let actualL1ExecutionFee = l1ExecutionFee
         .mul(Math.ceil(ethToSrcTokenPriceRatio * 1000))
         .div(1000);
 
+      if (sourceToken.decimals !== destToken.decimals) {
+        actualL1ExecutionFee = BigNumberUtils.convertTokenAmount(
+          formatUnits(actualL1ExecutionFee, destToken.decimals),
```

Comment:
It seems like we're formatting `actualL1ExecutionFee` and then immediately parsing it back. Is there a reason for this, or could we pass `actualL1ExecutionFee` directly to save a step?